File: app/src/main/java/spa/lyh/cn/singleweb/MainActivity.java
package spa.lyh.cn.singleweb;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import spa.lyh.cn.ft_webview.webview.WebViewActivity;

public class MainActivity extends AppCompatActivity {

    TextView tv;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        tv = findViewById(R.id.tv);
        tv.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, WebViewActivity.class);
                intent.putExtra("url","https://news.cctv.com/2022/10/19/ARTI1GmPVo7xLsTqOqsEStTM221019.shtml?spm=C94212.P4YnMod9m2uD.ENPMkWvfnaiV.34");
                //intent.putExtra("url","https://www.xindb.com/static/content/TGGJ/2021-10-26/app_902595663328849920.html");
                //intent.putExtra("url","https://umsweb.offshoremedia.net/login");
                intent.putExtra("title","测试标题");
                intent.putExtra("ua","cnsAndroid");
                startActivity(intent);
            }
        });
    }
}

File: app/src/main/java/spa/lyh/cn/singleweb/MainActivity.kt
package spa.lyh.cn.singleweb

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.view.View.OnClickListener
import androidx.appcompat.app.AppCompatActivity
import spa.lyh.cn.ft_webview.webview.WebViewActivity
import spa.lyh.cn.singleweb.databinding.ActivityMainBinding

class MainActivity: AppCompatActivity(),OnClickListener {
    private lateinit var b:ActivityMainBinding



    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        b = ActivityMainBinding.inflate(layoutInflater)
        setContentView(b.root)
        b.tv.setOnClickListener(this)
    }

    override fun onClick(p0: View?) {
        var intent:Intent
        when(p0?.id){
            R.id.tv ->{
                intent = Intent(this,
                    WebViewActivity::class.java)
                intent.putExtra("url","https://news.cctv.com/2022/10/19/ARTI1GmPVo7xLsTqOqsEStTM221019.shtml?spm=C94212.P4YnMod9m2uD.ENPMkWvfnaiV.34")
                //可以自定义第一次加载web的标题，并作为分享弹窗的触发条件之一。
                //注：分享弹窗是配合其他项目使用的，所以本项目本身并不带有shareDialog。
                //如果想启用这个功能只能按照我给出的shareDialog反射包名完全一致来写
                //所以这个功能本质可能没啥用，对你来说。
                intent.putExtra("title","测试标题")
                intent.putExtra("ua","coolAndroid")
                startActivity(intent)
            }
        }
    }
}